grits.sh

```diff
@@ -39,9 +34,14 @@ gzip -d grits.tar.gz
 docker load < grits.tar
 rm grits.tar
 
+#Start geonames api
+./geonames-api.sh
```

Comment:
This fails when the server's ethernet device is not named "eth0".. Looks like you can simply add `--ethernet $ethernet` to get it to work.